src/serac/physics/utilities/variational_form/detail/meta.h

```diff
@@ -0,0 +1,69 @@
+#pragma once

```

Comment:
I would rename this file `metaprogramming_utils` or something like that. Also, is there a reason this is `.h` instead of `.hpp`?

Reply:
Maybe just `metaprogramming.h`?